C/capnproto/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "capnp"
```

Comment:
@giordano I was debating whether the name should be `capnproto` or `capnp` all the build products are just `capnp`

Reply:
[Eh eh](https://github.com/JuliaPackaging/Yggdrasil/pull/319#discussion_r360652136).  I think we usually use the name of project, whatever the products are.  I did a quick research: [archlinux](https://www.archlinux.org/packages/community/x86_64/capnproto/) and [Debian](https://packages.debian.org/sid/capnproto) both use capnproto

Reply:
Lol, wonderful race condition.